fight.c

```diff
@@ -4119,14 +4120,14 @@ void do_bash( CHAR_DATA *ch, char *argument )
 
     ch->move -= (ch->level / 30);
 
-    if(is_affected(victim,gsn_precognition) && number_percent() > 50)
+ /*   if(is_affected(victim,gsn_precognition) && number_percent() > 50)
```

Comment:
Is there a reason this is commented out instead of deleted? Are we planning to reuse the code in a future PR? If not, please prune it instead of commenting. If we need to bring it back, that is the exact purpose of keeping a git history.